packages/gatsby-1-config-extract-plugin/package.json

```diff
@@ -8,7 +8,7 @@
   },
   "dependencies": {
     "babel-runtime": "^6.26.0",
-    "extract-text-webpack-plugin": "^3.0.2"
+    "extract-text-webpack-plugin": "^1.0.1"
```

Comment:
Installed the wrong version by mistake in the initial publishing - this version is compatible with Webpack 1.